alf/algorithms/sac_algorithm.py

```diff
@@ -620,8 +620,8 @@ def _critic_train_step(self, exp: Experience, state: SacCriticState,
         target_critic = target_critics.reshape(exp.reward.shape)
         entropy_reward = nest.map_structure(lambda la, lp: torch.exp(la) * lp,
                                             self._log_alpha, log_pi)
-        entropy_reward = sum(nest.flatten(entropy_reward))
-        target_critic = (target_critic - entropy_reward).detach()
+        entropy_reward = -sum(nest.flatten(entropy_reward))
```

Comment:
It might be better to move the negative sign up one more step into the lambda function in line 621,
Or can change the name of ``entropy_reward`` in Line 621 ?